src/pybind/mgr/cephadm/services/osd.py

```diff
@@ -647,6 +648,9 @@ def __init__(self,
         # The name of the node
         self.hostname = hostname
 
+        # is node OSD is deployed on offline
+        self.host_offline = host_offline
```

Comment:
I am often skeptical of "double negatives" - something like `no_status=False` vs. `status=True` to get status, especially in Python where you can default a kwarg to anything (like True).  This a bit more borderline because "offline-ness" could be seen as a "positive" value. I just ask to find out if you considered having it be online=True by default. :-)

Reply:
My thinking was offline is the positive value because the expected value is online and offline would be the abnormality.